.github/workflows/GoogleJavaFormat.yml

```diff
@@ -11,10 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: GoogleJavaFormat
```

Comment:
@sfc-gh-afarra since you made the original change, let us know if you have any concern